packages/easy-coding-standard/packages/SniffRunner/Application/SniffFileProcessor.php

```diff
@@ -81,6 +81,9 @@ public function processFile(SmartFileInfo $smartFileInfo, Configuration $configu
         $errorsAndDiffs = [];
 
         $file = $this->fileFactory->createFromFileInfo($smartFileInfo);
+
+        $file->setReportWarnings($configuration->getReportWarnings());
+
```

Comment:
Not needed

Reply:
This PR is to allow not listed warnings to be manages as errors, set the `File` object with the list of warnings is required, otherwise I don't know how to instruct `File`.

Reply:
These should be only listed